novawallet/Common/Protocols/Ramp/RampPresentable.swift

```diff
@@ -37,8 +37,18 @@ extension RampPresentable {
         ) else {
             return
         }
-        rampView.controller.modalPresentationStyle = .fullScreen
-        view?.controller.present(rampView.controller, animated: true)
+
+        if view?.controller.isBeingPresented == true {
```

Comment:
Didn't get why presentation method depends on whether current controller presentation is in progress or completed. It might be a typo so and we just wanted to check whether current controller is modally presented.